test/test.js

```diff
@@ -36,20 +34,4 @@ bedrock.events.on('bedrock.init', async () => {
   handlers.setUseHandler({handler: ({meter} = {}) => ({meter})});
 });
 
-bedrock.events.on('bedrock.ready', async () => {
```

Comment:
Why is it ok to remove all this stuff?

@gannan08, can you confirm that this shouldn't be here?

Reply:
This was what was suggested by @gannan08 to remove. I am sure he can confirm however the test suite would not run without this code removed due to the new meter changes.